.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Linux scripts always need to have lf
+*.sh text eol=lf
```

Comment:
Running the certificate generation script was failing for me because it had CRLF line endings, and Bash doesn't handle the "CR" part. Must use LF only.